### PR TITLE
dev, v3.0, v2.1: add more items in loader config template

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? 

loader config template is not up to date. I will update the Chinese doc after this is merged

### What is the related PR or file link(s)? 

Add more config items.

### Which version does your change affect? 

v3.0, v2.1, dev

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @csuzhangxc @TomShawn

File: dev/reference/tools/loader.md [hunk 1]
```diff
@@ -97,6 +97,14 @@ user = "root"
 password = ""
 port = 4000
 
+# sql_mode used when loading data. If not provided or set to "@DownstreamDefault", downstream global sql_mode will be used
```
Comment (TomShawn) [inline dev/reference/tools/loader.md]: If what is not provided or set to "@DownstreamDefault"? downstream global sql_mode?

File: dev/reference/tools/loader.md [hunk 2]
```diff
@@ -97,6 +97,14 @@ user = "root"
...
+# sql-mode = ""
+
+# max-allowed-packet sets max_allowed_packet for database connection,
```
Comment (TomShawn) [inline dev/reference/tools/loader.md]:
```suggestion
# max-allowed-packet sets `max_allowed_packet` for database connection.
```

File: dev/reference/tools/loader.md (hunk 1 above)
Comment (amyangfei) [reply dev/reference/tools/loader.md]: addressed, PTAL

File: dev/reference/tools/loader.md [hunk 3]
```diff
@@ -97,6 +97,14 @@ user = "root"
 password = ""
 port = 4000
 
+# sql_mode used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global sql_mode will be used
```
Comment (TomShawn) [inline dev/reference/tools/loader.md]: Are `sql_mode` and `sql-mode` used interchangeably? If not, please frame `sql_mode` with back quotes.

Comment (amyangfei) [reply dev/reference/tools/loader.md]: `sql_mode` is used in TiDB, `sql-mode` is used in  loader config. Both of them are added back quotes wrapping

**Pull request merged**